packages/compass-user-data/src/user-data.ts

```diff
@@ -12,14 +12,19 @@ type SerializeContent<I> = (content: I) => string;
 type DeserializeContent = (content: string) => unknown;
 type GetFileName = (id: string) => string;
 
-export type UserDataOptions<Input> = {
+export type FileUserDataOptions<Input> = {
   subdir: string;
```

Comment:
The naming on this one is unfortunate, but I think it's also relevant for the shared userdata interface